delete.go

```diff
@@ -17,6 +23,15 @@ status of "ubuntu01" as "destroyed" the following will delete resources held for
 	Action: func(context *cli.Context) {
 		container, err := getContainer(context)
 		if err != nil {
+			if lerr, ok := err.(libcontainer.Error); ok && lerr.Code() == libcontainer.ContainerNotExists {
+				// if there was an aborted start or something of the sort then the container's directory could exist but
+				// libcontainer does not see it because the state.json file inside that directory was never created.
+				path := filepath.Join(context.GlobalString("root"), context.Args().First())
+				if _, err := os.Stat(path); err == nil {
+					os.RemoveAll(path)
```

Comment:
I thought about that but I really didnt want to call removeAll before making sure that this existed.   What do you think?